src/content/RepoPage/RepoPage.js

```diff
@@ -1,5 +1,67 @@
-import React from 'react';
+import React, { useState } from 'react';
 import RepoTable from './RepoTable';
+import { gql } from 'apollo-boost';
+import { Query } from 'react-apollo';
+import { Link, DataTableSkeleton, Pagination } from 'carbon-components-react';
+
+const REPO_QUERY = gql`
+  query REPO_QUERY {
+    # Let's use carbon as our organization
+    organization(login: "carbon-design-system") {
+      # We'll grab all the repositories in one go. To load more resources
+      # continuously, see the advanced topics.
+      repositories(first: 75, orderBy: { field: UPDATED_AT, direction: DESC }) {
+        totalCount
+        nodes {
+          url
+          homepageUrl
+          issues(filterBy: { states: OPEN }) {
+            totalCount
+          }
+          stargazers {
+            totalCount
+          }
+          releases(first: 1) {
+            totalCount
+            nodes {
+              name
+            }
+          }
+          name
+          updatedAt
+          createdAt
+          description
+          id
+        }
+      }
+    }
+  }
+`;
+
+const LinkList = ({ url, homepageURL }) => (
```

Comment:
The case of the var `homepageUrl` does not match the definition in the REPO_QUERY query.